integration/crds/lightweight/DanmNet.yaml

```diff
@@ -1,116 +1,126 @@
-                  type: object
+                type: array
+              NetworkID:
+                type: string
+              NetworkType:
+                type: string
+              Options:
+                properties:
+                  alloc:
+                    description: bit array of tracking address allocation
+                    type: string
+                  alloc6:
+                    description: bit array tracking IPv6 allocations
+                    type: string
+                  allocation_pool:
+                    description: subset of the IPv4 subnet from which IPs can be allocated
+                    properties:
+                      end:
+                        type: string
+                      lastIp:
+                        type: string
+                      start:
+                        type: string
+                    required:
+                    - end
+                    - start
+                    type: object
+                  allocation_pool_v6:
+                    description: subset of the IPv6 subnet from which IPs can be allocated
+                    properties:
+                      cidr:
+                        type: string
+                    required:
+                    - cidr
+                    type: object
+                  cidr:
+                    description: IPv4 specific parameters IPv4 network address
+                    type: string
+                  container_prefix:
+                    description: The name of the interface in the container
+                    type: string
+                  device_pool:
+                    description: The resource_pool contains allocated device IDs
+                    type: string
+                  host_device:
+                    description: The device to where the network is attached
+                    type: string
+                  net6:
+                    description: IPv6 specific parameters IPv6 unique global address
+                      prefix
+                    type: string
+                  routes:
+                    additionalProperties:
+                      type: string
+                    description: IPv4 routes for this network
+                    type: object
+                  routes6:
+                    additionalProperties:
+                      type: string
+                    description: IPv6 routes for this network
+                    type: object
+                  rt_tables:
+                    description: Routing table number for policy routing
+                    type: integer
+                  vlan:
+                    description: the VLAN id of the VLAN interface created on top
+                      of the host device
+                    type: integer
+                  vxlan:
+                    description: the vxlan id on the host device (creation of vxlan
+                      interface)
+                    type: integer
+                required:
```

Comment:
same

Reply:
you can explain this too ???

Reply:
you removed validation rules which were there in the original API spec, therefore you made a backward incompatible change during the upgrade